app/uk/gov/hmrc/transitmovementsvalidator/controllers/MessagesController.scala

```diff
@@ -76,13 +109,20 @@ class MessagesController @Inject() (cc: ControllerComponents, xmlValidationServi
           .asPresentation
           .toValidationResponse
           .fold(
-            presentationError => Status(presentationError.code.statusCode)(Json.toJson(presentationError)(PresentationError.presentationErrorWrites)),
+            failCase,
             {
               case Some(r) => Ok(Json.toJson(r))
               case None    => NoContent
             }
           )
-
     }
 
+  private def getObjectStoreUri(implicit request: RequestHeader) = EitherT {
+    Future.successful(
+      request.headers
+        .get("X-Object-Store-Uri")
+        .toRight(PresentationError.unsupportedMediaTypeError("Content Type or X-Object-Store-Uri must be specified."))
```

Comment:
I think this should be bad request error and also we are not supplying any content type so should not include ``` Content Type ```

Reply:
I did have that as a bad request previously but changed it to handle the no content type.

Reply:
I will follow the same as we have transit-movements.